gem/refactorise.py

```diff
@@ -229,6 +230,10 @@ def collect_monomials(expressions, classifier):
                                       predicate=lambda i: i in must_unroll)
         expressions = remove_componenttensors(expressions)
 
+    # Expand Conditional nodes which are COMPOUND
+    conditional_predicate = lambda node: classifier(node) == COMPOUND
```

Comment:
There is a rule against assigning `lambda`s to variables. But I guess we ignore that then.